lib/app/item.ex

```diff
@@ -2,23 +2,26 @@ defmodule App.Item do
   use Ecto.Schema
   import Ecto.Changeset
   import Ecto.Query
-  alias App.{Repo, Tag, ItemTag, Person}
+  alias App.{Repo, Tag, ListItem, ItemTag, Person}
+  alias App.List, as: L
   alias __MODULE__
 
   schema "items" do
     field :status, :integer
     field :text, :string
+    field :item_lists, {:array, :string}, virtual: true
```

Comment:
Not a fan of this. But we can keep it for now. 👍

Reply:
This is a custom field not used in Postgres to make it easier to work with the checkbox group component from Petal:
![image](https://user-images.githubusercontent.com/6057298/194833511-6d490a69-c0d7-4ef8-a921-55043a9489a5.png)
https://petal.build/components/forms#checkbox_group

This is to represent the `options` value for the checkbox and to select the lists we want the item to belongs to:
```elixir
<.form_field
  type="checkbox_group"
  form={f}
  field={:your_field}
  label="Column layout"
  options={[
    {"Option 1", "option_1"},
    {"Option 2", "option_2"},
    {"Option 3", "option_3"},
    {"Option 4", "option_4"}
  ]}
/>
```
In the example we can see the options is a list of tuple and that's why I've defined the `:item_lists` as a list of string (I need to double check if this should even be a list of tuple of string)